### DISCUSSION TITLE
推荐外挂增强脚本，解决原文遮挡问题。

### DISCUSSION THREAD

**Discussion by KnIfER** [crimx/ext-saladict#1226] [Show and tell]
沙拉查词的默认弹出面板会遮蔽原文，所以很多情况下需要用鼠标将查词面板移动来、移动去，造成阅读不便。

（当然独立窗口查词加大屏幕或可无视此问题，不在讨论范围内）

故分享一个脚本调节器，可动态调节透明度，默认的查词面板近乎全透明，不会影响原文阅读，当鼠标移到查词面板之上、处于“悬浮”状态之时，才会迅速提高不透明度，继而允许正常阅读翻译结果，使用体验流畅无比！

安装地址（油猴脚本）：https://greasyfork.org/zh-CN/scripts/421242-saladict-transparency-tweaker

**Comment by KnIfER** [comment]
或可以此为范式，让用户自定义沙拉查词的外观，而作者君只需专注打磨内核即可！

**Reply by pickalexrener** [reply]
链接失效了，可以重新分享一下吗？谢谢。

**Comment by crimx** [comment]
感谢你的分享！

> 让用户自定义沙拉查词的外观

沙拉查词设置中可以自定义 CSS 样式，不知道能不能达到这个效果。
